dockers/docker-orchagent/docker-init.j2

```diff
@@ -3,7 +3,7 @@
 mkdir -p /etc/swss/config.d/
 mkdir -p /etc/supervisor/
 mkdir -p /etc/supervisor/conf.d/
-
+mkdir -p /dev/shm/supervisor/
```

Comment:
Is orchagent the only container that needs this? Or is it handled for the other containers by some other means?

Reply:
Orchagent was the only container we found was contributing towards disk writes. I'm hesitant to "fix it if it aint broke", so to speak.

Reply:
made /var/log/supervisor directory tmpfs on all containers after internal discussion